### ISSUE TITLE
Removing StockItem results in "Form Error - Errors exist for one or more form fields"

### ISSUE THREAD

**Issue by simonkuehling** [inventree/InvenTree#8025]
### Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find a similar issue

### Describe the bug*

On one specific stock item in the warehouse I get an error when trying to reduce the stock count ("remove stock") via the PUI interface. The same action works without issues through the iOS app, though - and other stock items can be removed successfully as well.

This is the error in the modal:
![grafik](https://github.com/user-attachments/assets/31bc5db9-54db-4661-8274-dd616190fb0e)

The raw request is
```
{"items":[{"pk":735,"quantity":1,"batch":"","status":10,"packaging":null,"obj":{"batch":"","belongs_to":null,"build":null,"consumed_by":null,"customer":null,"delete_on_deplete":false,"expiry_date":null,"is_building":false,"link":"","location":94,"location_name":"C1.2","location_detail":{"pk":94,"name":"C1.2","pathstring":"Lager/C/C1/C1.2"},"notes":null,"owner":null,"packaging":null,"parent":null,"part":302,"part_detail":{"pk":302,"IPN":"302","barcode_hash":"","category_default_location":null,"default_location":null,"name":"Zylinderschraube, mit Innensechskant, Stahl verzinkt - M8x60","revision":"","full_name":"302 | Zylinderschraube, mit Innensechskant, Stahl verzinkt - M8x60","description":"ISO 4762 (= DIN 912), Gewinde M8x1.25, hex socket head cap screw","image":"/media/part_images/part_285_image.png","thumbnail":"/media/part_images/part_285_image.thumbnail.png","active":true,"locked":false,"assembly":false,"component":true,"is_template":false,"purchaseable":true,"salable":false,"testable":false,"trackable":false,"virtual":false,"units":"","pricing_min":"0.486000","pricing_max":"0.680500"},"purchase_order":9,"purchase_order_reference":"PO-0009","pk":735,"quantity":19,"sales_order":null,"serial":null,"status":10,"status_text":"OK","stocktake_date":null,"supplier_part":216,"sku":"2524-118","supplier_part_detail":{"available":0,"availability_updated":null,"description":"","link":"https://www.wegertseder.com/ISO-4762-Zylinderschrauben-mit-Innensechskant-STAHL-8-8-verzinkt","active":true,"manufacturer_part":null,"name":"302 | Wegertseder GmbH | 2524-118","note":"","pk":216,"barcode_hash":"","packaging":"","pack_quantity":"20","pack_quantity_native":20,"part":302,"SKU":"2524-118","supplier":52,"url":"/supplier-part/216/","updated":"2023-07-11 08:54","notes":null,"tags":[]},"barcode_hash":"","updated":"2024-08-28 13:48","purchase_price":"0.486000","purchase_price_currency":"EUR","allocated":0,"expired":false,"installed_items":0,"child_items":0,"stale":false,"tracking_items":3,"tags":[]}}]}
```

the response is
```
{"items":[{"packaging":["This field may not be null."]}]}
```

### Steps to Reproduce

1. Open the stock item
2. Stock Action "Remove"
3. enter a number and click "submit"

### Expected behaviour

The "packaging" field in the supplier part for that stock item is empty - like with almost all of our stock items. Not sure in which way this differs here... any further data I can collect for debugging?

### Deployment Method

- [X] Docker
- [ ] Package
- [ ] Bare metal
- [ ] Other - added info in Steps to Reproduce

### Version Information

InvenTree Version	
[0.17.0 dev](https://github.com/inventree/InvenTree/releases)
Commit Hash	
5515e07
Commit Date	
2024-08-20
Commit Branch	
API Version	
[242](http://192.168.177.12/api-doc/)
Python Version	
3.11.9
Django Version	
[4.2.15](https://www.djangoproject.com/)

### Please verify if you can reproduce this bug on the demo site.

- [ ] I can reproduce this bug on the demo site.

**Comment by SchrodingersGat** [comment]
@simonkuehling I think this bug was fixed in https://github.com/inventree/InvenTree/pull/7981 - please download latest code and confirm

**Comment by simonkuehling** [comment]
@SchrodingersGat confirmed - this seems to be fixed (running 3911694 now)!

thanks for the hint..
